app/states/texas/models.py

```
@@ -0,0 +1,64 @@
+from typing import Optional
+from datetime import date
+from pydantic import ConfigDict, model_validator, BaseModel, BeforeValidator
+from sqlmodel import SQLModel, Field
+from .validators.texas_settings import TECSettings
+import app.states.texas.funcs.tx_validation_funcs as tx_funcs
+import app.funcs.validator_functions as funcs
+from app.funcs.record_keygen import RecordKeyGenerator
+
+class TECFlags(TECSettings, table=True):
+    __tablename__ = "tx_flags"
+    __table_args__ = {"schema": "texas"}
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)